src/components/PatronBlock/PatronBlock.test.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import {
+  render,
+  screen,
+} from '@testing-library/react';
+
+import '../../../test/jest/__mock__';
+
+import PatronBlock from '.';
+
+describe('PatronBlock', () => {
+  const testIds = {
+    patronBlock: 'patronBlock',
+    patronBlockLink: 'patronBlockLink',
+    patronBlockLabel: 'patronBlockLabel',
+    warnIcon: 'warnIcon',
+  };
+  const labelIds = {
+    label: 'ui-checkout.patronBlocks',
+    patronMessage: 'ui-checkout.patronBlocksCount',
+  };
+  const renderComponent = (props) => {
+    const combined = {
+      patronBlocksCount: 1,
+      user: {
+        id: 0,
+      },
+      ...props,
+    };
+
+    render(<PatronBlock {...combined} />);
+  };
+
+  describe('when patronBlocks > 0', () => {
+    beforeEach(() => renderComponent());
+
+    it('should render container div', () => {
```

Comment:
Could you please investigate should we rename `it` from `should render container div `to `should render patronBlock container`?